static/js/selectors/websites.ts

```diff
@@ -8,8 +8,21 @@ import { ReduxState } from "../reducers"
 import { WebsiteStarter } from "../types/websites"
 
 export const getWebsiteListingCursor = createSelector(
-  (state: ReduxState) => state.entities?.websitesListing ?? {},
-  websitesListing => memoize((offset: number) => websitesListing[offset])
+  (state: ReduxState) => ({
+    listing: state.entities?.websitesListing ?? {},
+    details: state.entities?.websiteDetails ?? {}
+  }),
+  lookup =>
+    memoize((offset: number) => {
+      const response = lookup.listing[offset] ?? {}
+      const names = response?.results ?? []
+      const sites = names.map((name: string) => lookup.details[name])
+
+      return {
+        ...response,
+        results: sites
+      }
+    })
```

Comment:
this looks fine to me I think, you could also use the `getWebsiteDetailCursor` as an input selector here if you wanted, but that wouldn't make it all *that* much more simple (just perhaps a bit more elegant, you could then do something like `names.map(cursor)` I think)

Reply:
I made an attempt though I'm uncertain that it's an improvement. What do you think? Here's the diff: https://github.com/mitodl/ocw-studio/pull/108/commits/ac1c55c3841619205a149133dffae73235b569c8